wagtail/admin/templates/wagtailadmin/shared/header_with_locale_selector.html

```diff
@@ -1,6 +1,8 @@
 {% extends "wagtailadmin/shared/header.html" %}
 
-{% block right_column_classname %}right col{% endblock %}
+<!-- No longer no longer supported in the release notes 
+    {% block right_column_classname %}right col{% endblock %}
+ -->
```

Comment:
This is not a suitable solution to the block being removed, we need to remove this line completely and see whether any styles are broken and how to best fix them in a generic way.